### ISSUE TITLE
YT Comments module doesnt return csv files

### ISSUE THREAD

**Issue by maxpi99** [bernorieder/YouTube-Data-Tools#10]
Hello DMI-team,

I believe there may be an issue with the comments module. I attempted to scrape a video with approximately 60,000 comments, but it seems that the process stops at the end, and the web tool does not provide the CSV files for the video. The provided screenshot shows the end of the page after the process finished with the extraction of the comments.
![grafik](https://github.com/bernorieder/YouTube-Data-Tools/assets/98280131/84e19763-7c2f-4459-bed4-8c4bd8059c2b)

**Comment by bernorieder** [comment]
please read the FAQ for how to report a problem in a way that allows us to help

**Comment by maxpi99** [comment]
Sorry, the video id is "Yve9DhT8Nt4". I used the Firefox browser to scrape a video with about 67k comments. After the processing seems to be finished the webtool doesnt show/produce the data files (Basic info, Authors, comments). (I tried it also with a different video with a similar comment count but it also doesnt show/produce the data files.

Here is the Link: https://ytdt.digitalmethods.net/mod_video_comments.php?videohash=Yve9DhT8Nt4&toplimit=&output=csv&g-recaptcha-response=03AFcWeA58R07qY0m6jtgaikJ4FiUpOflERYbewYmG4YOXzsI-kmRq166HR2WcuUwGCQlam1XzpcPq53KX1sPYir91cetQXaLa2uZGJohZifCw8ZJSDLCywgioPa4zFcLy7zjM7rKYjAdfU-jpf0N6Qowa2kSzPfTKuHazwnsC0U6Kxepf8lhBqIsGsEAnghcwlGc9EsLGRisOxrtkp841OMXdANw8Xs9DU6XpbeVglZJEj2i7_o77QBc5JG6joXi9eFhM1qSiwOB_zjUxX9cjGhRepQYYS3yawypqdS5D1PRVJDfgeWtn7d3oagV4NbzBsAe84LwQgpCmMgG5LMSfHOZLfF-JjImqBPADTp8cbhmXlB8f3VkESkhisgApJkZBXIOPg-gXxsq2lJ0_FOGYZFpDjIVzWph-BQqosfPTKeDC96yV2zFjTR2jQAW-FePbHB3hQmmEh367KJmKAep6q2lvnYXr7fpQj4X2aglQX3vudjA6krXeF-jYut5XuDNvI0qPNiWHe5XLaml3lzJxcXz7NrwZpTPcDvCtwXotAV_P4kzUg9-kPhoaQ7JzUrOunPevd6f02AB51FAx0QgSRUZCBgB9sAS4p-iAqRNiy4eFLD1Yi2mtYuiEUbCqQy8zbo7VST5ZHkROME6DRlghlVfFPOC51ZlYmkdMAwlEM2A-53ZiIio-vtJ5GycsX_qeTh-Dg3Bhxti7CXdTNxqXZgPqPaj5ZjZ7aofqDSx5Imx2zgpiV3g3d1bQrj_iUzzCLME8mKL2hKlJU_Ru-UJl3qOddIpEbrcToouv_m-YXtKfzSO9LmU8xpdn00Xx1D6q4g5G98ph73tBzT7ZSpamCJO5ljJTz46m3_ndmR16AGmm3duE2F20YmfZPAaDssKX8PgUtBj_Z8ooWN3smU7IAkuLVQTi3ZtkyYb2951dxDH5Ac_Zdjps-LCPzLgN3CmTO5_H9aEOFczN9hldRX4-bk1U_EvxMCrxXJSWPAvMoElrrUhRkF1xhvx52fLVL4KMvLKJCS5E3DUWEz5aOfW4Jj8Z6Zm58QuFPkETUGG8ClHSbZ4bqibKvuIYaJwzNOc8WHpea7IuyQzfyujnV42JKTJJKdvAPKy3glmE60qvIkQOD2tzmB0Gcgu07o2qesMnZfQiVb5spaOnItBYQ-2pKSRzvQ0fwVULbYrtHgn-H3GU0v2171ze4aA6CWT3AkfNakoMr9eGqBmMWQpYyv8o18g9Ts-UgSSkA86Po7WSu5debDo7BIZF_z_urDi3Gc0Lna2PhK2b8DkcB5kM1n4dE383bc3zJ-vx2PKatPY3xE0DjKp6c3Hsf2wcFzAFzV8kdFBtUIJy17_sR9iC0mqykgvRaiMRQA7hb2mmfnrHG0VC5Y1ZfMMGsxjbgBLYYARVBzQ4Ntqi5EQnvvcHdMwCRpBrf8GvKjShoEaGLx29_NCsViaZvbgKnGloxyUiCjywdr7Jass1oB_nQB13dOV2aP2efQGk8WjcY29vji5_88oh1RxrbSdToAUaLRBtUvWClwwRj1YknZ5DDIrlEwcIrgKZIVrInhaEIrUTyIQPwhNBhwY_H7bA9oLLcRsZ202fcsGxHq98ZjfyrSdcvfxi4ui3UZXXN11MEloze4c_vZceKqvKV8gQINbL1r6dggQrQS14vhZeSebLFfIXxpx4AlTdWwQcLEWGChoIos5vlPuNsebjbCDDPFzX9A43dsKUbZSgai34uPU2B1Gr_W3mak6DhBc2xYqSQg

<img width="951" alt="foto2" src="https://github.com/bernorieder/YouTube-Data-Tools/assets/98280131/c8a953e0-b283-4957-a519-9bb73029adab">
<img width="968" alt="foto1" src="https://github.com/bernorieder/YouTube-Data-Tools/assets/98280131/332d5e18-ce12-431d-ad0f-affe697f9b51">
